### PR TITLE
Add support for PS3 Keyboard

### PR DESCRIPTION
**Pull request by alkyl-twists**
Init code taken from what RPCS3 sends to the keyboard when launching rb3. Without sending, the keyboard doesn't return any (keyboard) keys.

Handle key changed so previous state is used for noteup and notedown. Was getting noteup for every key not pressed everytime in aseqdump.

### CODE REVIEW

**Comment by jasperro** [comment]
I just tried this new commit out, but now the noteups don't seem to register at all, when you press a note on the RB3 xbox360 keyboard, it won't stop playing. Reverting to before this change seems to fix it. Any idea why this could be?

File: src/rb3keyboard.c [hunk 1]
```diff
@@ -121,12 +145,12 @@ static inline void handle_key(MIDIDRUM *MIDI_DRUM, unsigned char key) {
...
+        } else {
+            MIDI_DRUM->noteup(MIDI_DRUM->sequencer, MIDI_DRUM->channel,
+                                                    48 + key + 12 * MIDI_DRUM->octave,
+                                                    0);
         }
     }
-    if (!(MIDI_DRUM->key_state & (1 << (24 - key)))) {
-        MIDI_DRUM->noteup(MIDI_DRUM->sequencer, MIDI_DRUM->channel,
-                                            48 + key + 12 * MIDI_DRUM->octave, 0);
-    }
```
Comment (jasperro) [inline src/rb3keyboard.c:129]: Adding back these lines seems to fix the issue, why was this removed, was there a specific reason?

**Review by jasperro — COMMENTED** [review]
@alkyl-twists Adding back these lines seems to fix the issue of noteups not being detected, why was this removed, was there a specific reason?

**Comment by ssj71** [comment]
It's been long enough that I'm  not sure why anymore. I don't own any keyboards and so I'm not really able to support this part of the code well by myself. If you are willing I'd be happy to accept any pull requests you can make to get the code working.

**Comment by jasperro** [comment]
@alkyl-twists I just made pull https://github.com/rbdrum2midi/rbdrum2midi/pull/13, could you test if this still works with your PS3 keyboard as you had some issues with noteup detection?

File: src/rb3keyboard.c (hunk 1 above)
Comment (alkyl-twists) [reply src/rb3keyboard.c:129]: I think I changed it as I was getting Note Off for every key (unpressed) constantly in aseqdump.

**Comment by alkyl-twists** [comment]
Pull 13 seems to work for me
